README.md

```diff
@@ -10,7 +10,8 @@ Now you can build even bigger and better apps and integrate them with your bank
 
 Before you dive into this brand new SDK, please consider:
 - Checking out our new developer’s page [https://bunq.com/en/developer](https://bunq.com/en/developer) 🙌  
-- Grabbing your production API key from the bunq app or asking our support for a Sandbox API key 🗝
+- Grabbing your production API key from the bunq app or generate a Sandbox API key using the
+[Tinker endpoint](https://together.bunq.com/topic/tinker-the-api) 🗝
```

Comment:
Use `..API key using tinker` instead of `the tinker endpoint`

Reply:
@OGKevin let's keep the discussion about the phrasing in one PR; I added endpoint on purpose. There's already a lot of confusion where people think Tinker is another SDK/API, while it's just an endpoint. Hence, I made it explicit here. Still want me to change it?